db/schema.rb

```diff
@@ -1,12 +1,18 @@
 ActiveRecord::Schema.define(version: 20160719122949) do
-  # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-
+  
```

Comment:
You've left some white-character in an empty line, delete the white-space characters. Even better, use some kind of linter or run `bundle exec rubocop` before pushing to catch such things.